src/js/shield_defs.js

```diff
@@ -3238,6 +3238,32 @@ export function loadShields() {
 
   // ASIA
 
+  // Asian Highway Network
+  shields["AsianHighway"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white
+  );
+  shields["AsianHighway"].overrideByName = {
+    green: roundedRectShield(Color.shields.green, Color.shields.white),
+    green_in: {
+      spriteBlank: ["shield_in_ah_2", "shield_in_ah_3", "shield_in_ah_4"],
+      textColor: Color.shields.white,
+      padding: {
+        left: 2,
+        right: 2,
+        top: 2,
+        bottom: 7,
+      },
+    },
+    white: roundedRectShield(Color.shields.white, Color.shields.black),
+    white_id: roundedRectShield(
+      Color.shields.white,
+      Color.shields.green,
+      Color.shields.black
+    ),
+  };
+  shields["AH"] = shields["AsianHighway"];
```

Comment:
[This wiki page](https://wiki.openstreetmap.org/wiki/Special:PermanentLink/2349783#Tags_for_the_relation) specifies `network=AsianHighway` for Asian Highway routes (along with `int_ref`, grr), but a significant minority of routes are tagged `network=AH` instead. I can’t find any discussion of using `AH` anywhere, so it’s probably just a lack of awareness about the `AsianHighway` standard. A two-letter code like `AH` would be problematic anyways, because it would be easily confused with a (nonexistent) ISO&nbsp;3166-1 country code.

Reply:
Sounds like a strong case against supporting `network=AH`. I would rather leave it out to encourage mappers to use a standard tagging scheme.

Reply:
Indeed, I’m definitely planning to remove support for it before merging. For now, though, it is necessary for demonstrating some of the cross-border differences in shield designs, because they happen to coincide with cross-border differences in how some route relations have been tagged. (I suspect it’s an oversight rather than anything intentional.)